code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -106,9 +106,12 @@
 /datum/chem_property/positive/hemogenic/process(mob/living/M, potency = 1, delta_time)
 	if(!iscarbon(M))
 		return
-	var/mob/living/carbon/C = M
-	C.blood_volume = min(C.blood_volume+potency,BLOOD_VOLUME_MAXIMUM+100)
-	if(potency > POTENCY_MAX_TIER_1 && C.blood_volume > BLOOD_VOLUME_MAXIMUM && !isyautja(M)) //Too many red blood cells thickens the blood and leads to clotting, doesn't impact Yautja
+	if(M.nutrition < 200)
+		return
+
+	M.nutrition -= 5
+	M.blood_volume = min(M.blood_volume + potency, M.limit_blood)
+	if(potency > POTENCY_MAX_TIER_1 && M.blood_volume > (M.max_blood + 10) && !isyautja(M)) //Too many red blood cells thickens the blood and leads to clotting, doesn't impact Yautja
```

Comment:
if blood is not suposed to go over maximun then this is not relevant anymore

Reply:
Max_blood is actually just 100% while limit_blood is like 110% or something and the hard limit.